content/zh/docs/faq/multi-cluster-management/manage-multi-cluster.md

```diff
@@ -1,61 +1,60 @@
 ---
-title: "Manage a Multi-cluster Environment on KubeSphere"
-keywords: 'Kubernetes, KubeSphere, federation, multicluster, hybrid-cloud'
-description: 'Understand how to manage a multi-cluster environment on KubeSphere.'
-linkTitle: "Manage a Multi-cluster Environment on KubeSphere"
+title: "在 KubeSphere 上管理多集群环境"
+keywords: 'Kubernetes，KubeSphere，联邦，多集群，混合云'
+description: '理解如何在 KubeSphere 上管理多集群环境。'
+linkTitle: "在 KubeSphere 上管理多集群环境"
 weight: 16710
----
-
-KubeSphere provides an easy-to-use multi-cluster feature to help you [build your multi-cluster environment on KubeSphere](../../../multicluster-management/). This guide illustrates how to manage a multi-cluster environment on KubeSphere.
 
-## Prerequisites
+---
 
-- Make sure your Kubernetes clusters are installed with KubeSphere before you use them as your Host Cluster and Member Clusters. 
+KubeSphere 提供了易于使用的多集群功能，帮助您[在 KubeSphere 上面构建多集群环境](../../../multicluster-management/)。本指南将会说明如何在 KubeSphere 上面管理多集群环境。
```

Comment:
I prefer put a "将” before “说明” to enhance the fluency of this sentence. Would you mind discuss it?

Reply:
Sure. I see your point. In technical writing, it is better to state the fact in the context of a document. `将` implies a future tense while our readers can get the information described in this document with nothing to expect in the future.
This is a matter of style instead of a matter of right or wrong.

Reply:
I got it, thanks.